packages/react/src/components/ErrorBoundary.tsx

```diff
@@ -7,10 +7,11 @@ type ErrorComponentProps = {
   [prop: string]: unknown;
 };
 
+// @TODO: remove type in next major release
 export type ErrorBoundaryProps = {
   children: ReactNode;
   page: Page;
-  type: string;
+  type?: string;
```

Comment:
is error boundary exported from core package, maybe we can just remove it?
If not, it would be great to mark it as obsolete with tsdoc